playbooks/roles/nginx/files/sites/cardstories/cardstories.org.conf

```diff
@@ -0,0 +1,70 @@
+server {
```

Comment:
@Agrendalath, is this needed any more? I am not sure if it is used. Also, the cardstories.org site uses an expired SSL certificate